feature-libs/checkout/base/components/checkout-shipping-address/checkout-shipping-address.component.ts

```diff
@@ -168,24 +172,33 @@ export class CheckoutShippingAddressComponent implements OnInit {
   }
 
   addAddress(address: Address | undefined): void {
-    if (address) {
```

Comment:
👍  was thinking of flattening later after we fix the issue, but coolio that you did it